deploy/scripts/src/accounts/vc-storage.ts

```diff
@@ -140,22 +148,86 @@ export function summariseVC(): void {
+  }
+
+  body = JSON.stringify([
+    generateCreateVCPayload(jwtTokens.fraud, 'PENDING'),
+    generateCreateVCPayload(jwtTokens.passport, 'PENDING'),
+    generateCreateVCPayload(jwtTokens.kbv, 'PENDING')
+  ])
+
+  timeGroup(groups[0], () => http.post(env.envURL + `/vcs/${subjectID}`, body, options), {
+    isStatusCode202: r => r.status === 202,
+    ...pageContentCheck('messageId')
+  })
+
+  sleep(1)
+
+  const jwtSignatures = {
+    fraud: 'MEQCIGmxCN7yShXAlwU5ZMim1fjVXOWgMgWfoiHnqW-wt6FGAiAL01oZxiu6n9FKpEZQXLUdxWQ-uSFrVb9J5PejaWHFUQ', // pragma: allowlist secret
+    passport: 'MEUCIQC4skxDIGh4eHyfA-5rNvLwlVIEgJVdVYaSfehI8T78SAIgJN1F52jDNY06D0RIhE_DtZT5h-zae1pBPZ3MDDtKgFI', // pragma: allowlist secret
+    kbv: 'MEUCIQC8U5VKnsYhlt35vMCaaBLws_3WqfDHMnCnRJIdJ7v_zQIgMdsi_B2UEsD7P_QwLsG7owv4eI_AV5oTpjtmZCeKhs8' // pragma: allowlist secret
+  }
+
+  body = JSON.stringify([
+    generateUpdateVCPayload(jwtSignatures.fraud, 'CURRENT'),
+    generateUpdateVCPayload(jwtSignatures.passport, 'CURRENT'),
+    generateUpdateVCPayload(jwtSignatures.kbv, 'CURRENT')
+  ])
+
+  timeGroup(groups[1], () => http.patch(env.envURL + `/vcs/${subjectID}`, body, options), {
+    isStatusCode202: r => r.status === 204,
+    ...pageContentCheck('messageId')
+  })
+
+  iterationsCompleted.add(1)
+}
+
 function getToken(r: Response): string {
   const token = r.json('token')
   if (token !== null && typeof token === 'string') return token
   fail('token not found')
 }
 
-interface VCPayload {
+interface CreateVCPayload {
   vc: string
   state: string
   provenance: string
 }
 
-function generateVCPayload(vcJWT: string): VCPayload {
+function generateCreateVCPayload(vcJWT: string, vcState: string): CreateVCPayload {
```

Comment:
```suggestion
function generateCreateVCPayload(vcJWT: string, vcState: VcState): CreateVCPayload {
```
```
type VcState = 'CURRENT' | 'PENDING'
```
Could make a type for the VC state to ensure it it always of the correct value

Reply:
Updated